sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/tests/PlaywrightReporterTests.cs

```diff
@@ -0,0 +1,266 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System;
+using Azure.Developer.MicrosoftPlaywrightTesting.TestLogger.Interface;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+using Moq;
+using Azure.Developer.MicrosoftPlaywrightTesting.TestLogger.Utility;
+
+namespace Azure.Developer.MicrosoftPlaywrightTesting.TestLogger.Tests
+{
+    [TestFixture]
+    internal class PlaywrightReporterTests
+    {
+        [TearDown]
+        public void TearDown()
+        {
+            Environment.SetEnvironmentVariable(ServiceEnvironmentVariable.PlaywrightServiceUri, null);
```

Comment:
should we do this in Setup as well to make sure variables are set or you think initializer should take care of it.

Reply:
Teardown will clear any variable that was set in the Environment